rcl/test/rcl/test_subscription.cpp

```diff
@@ -686,43 +957,92 @@ TEST_F(
   {
     EXPECT_EQ(RMW_RET_OK, rmw_message_info_sequence_fini(&message_infos));
   });
+  rmw_subscription_allocation_t * allocation = nullptr;  // is a valid argument
 
   EXPECT_EQ(
     RCL_RET_SUBSCRIPTION_INVALID,
-    rcl_take_sequence(nullptr, seq_size, &messages, &message_infos, nullptr));
+    rcl_take_sequence(nullptr, seq_size, &messages, &message_infos, allocation));
   rcl_reset_error();
   EXPECT_EQ(
     RCL_RET_SUBSCRIPTION_INVALID,
-    rcl_take_sequence(&subscription_zero_init, seq_size, &messages, &message_infos, nullptr));
+    rcl_take_sequence(&subscription_zero_init, seq_size, &messages, &message_infos, allocation));
   rcl_reset_error();
 
   EXPECT_EQ(
     RCL_RET_INVALID_ARGUMENT,
-    rcl_take_sequence(&subscription, seq_size + 1, &messages, &message_infos, nullptr));
+    rcl_take_sequence(&subscription, seq_size + 1, &messages, &message_infos, allocation));
   rcl_reset_error();
   EXPECT_EQ(
     RCL_RET_INVALID_ARGUMENT,
-    rcl_take_sequence(&subscription, seq_size, &messages, &message_infos_short, nullptr));
+    rcl_take_sequence(&subscription, seq_size, &messages, &message_infos_short, allocation));
+  rcl_reset_error();
+
+  EXPECT_EQ(
+    RCL_RET_INVALID_ARGUMENT,
+    rcl_take_sequence(&subscription, seq_size, nullptr, &message_infos, allocation));
   rcl_reset_error();
 
-  // This test fails for rmw_cyclonedds_cpp function rmw_take_sequence
-  // Tracked here: https://github.com/ros2/rmw_cyclonedds/issues/193
```

Comment:
Is this issue now resolved or does the test no longer fail?

Reply:
Issue is not resolved, but test no longer relies on it to work.